vumi/scripts/vumi_list_messages.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+# -*- test-case-name: vumi.scripts.tests.test_vumi_list_messages -*-
+
+import sys
+
+from twisted.internet.defer import inlineCallbacks
+from twisted.internet.task import react
+from twisted.python import usage
+
+from vumi.components.message_store import MessageStore
+from vumi.persist.txriak_manager import TxRiakManager
+
+
+class Options(usage.Options):
+    optParameters = [
+        ["batch", None, None,
+         "Batch identifier to list messages for."],
+        ["bucket-prefix", "b", None,
+         "The bucket prefix for the Riak manager."],
+        ["direction", None, None,
+         "Message direction. Valid values are `inbound' and `outbound'."],
+        ["index-page-size", None, "1000",
+         "The number of keys to fetch in each index query."],
+    ]
+
+    longdesc = """
+    Index-based message store lister. For each message, the timestamp, remote
+    address, and message_id are returned in a comma-separated format.
+    """
+
+    def ensure_dependent_option(self, needed, needs):
```

Comment:
Thanks. I copied it from elsewhere and then forgot to delete it when I deleted the things that used it.